src/app/component/header/header.component.html

```diff
@@ -64,8 +46,8 @@
 </nav>
 
 <div
-  class="modal fade"
-  id="exampleModal"
+  class="modal"
```

Comment:
why model ? as already told that no need for model

Reply:
I have Remove model